Cubical/Algebra/Polynomials/TypevariateHIT/EquivUnivariateListPoly.agda

```diff
@@ -0,0 +1,73 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.Polynomials.TypevariateHIT.EquivUnivariateListPoly where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Structure
+
+open import Cubical.Data.Unit
+
+open import Cubical.Algebra.CommRing
+open import Cubical.Algebra.Algebra
+open import Cubical.Algebra.CommAlgebra
+open import Cubical.Algebra.Polynomials.TypevariateHIT
+            renaming (inducedHomUnique to inducedHomUniqueHIT;
+                      isIdByUMP to isIdByUMP-HIT)
+open import Cubical.Algebra.Polynomials.UnivariateList.UniversalProperty
+            renaming (generator to generatorList;
+                      inducedHom to inducedHomList;
+                      inducedHomUnique to inducedHomUniqueList;
+                      isIdByUMP to isIdByUMP-List)
+
+private variable
+  ℓ : Level
+
+module _ {R : CommRing ℓ} where
+  open Theory renaming (inducedHom to inducedHomHIT)
+  open CommRingStr ⦃...⦄
+  private instance
+    _ = snd R
+    X-HIT = Construction.var {R = R} {I = Unit} tt
+    X-List = generatorList R
```

Comment:
These two should not be instances, or should they?

Reply:
Yes, they shouldn't, thanks!